README.md

```diff
@@ -27,62 +23,39 @@ Then perform the following commands on the config folder:
 - `terraform apply` to apply the infrastructure build
 - `terraform destroy` to destroy the built infrastructure
 
-[^]: (autogen_docs_start)
-
-
-## Inputs
-
-| Name | Description | Type | Default | Required |
-|------|-------------|:----:|:-----:|:-----:|
-| cloud_sql_region | The Cloud SQL region | string | `us-central1` | no |
-| credentials_file_path | Path to service account json | string | - | yes |
-| download_forseti | Whether to download the forseti repo or not. If false, a Forseti repo must be in the root of main.tf file. (Default 'true') | string | `true` | no |
-| forseti_repo_branch | Forseti repository branch | string | `stable` | no |
-| forseti_repo_url | Foresti git repository URL | string | `https://github.com/GoogleCloudPlatform/forseti-security.git` | no |
-| gcs_location | The GCS bucket location | string | `us-central1` | no |
-| gsuite_admin_email | The email of a GSuite super admin, used for pulling user directory information. | string | - | yes |
-| notification_recipient_email | Notification recipient email | string | - | yes |
-| project_id | The ID of the project where Forseti will be installed | string | - | yes |
-| sendgrid_api_key | The Sendgrid api key for notifier | string | `` | no |
-
-## Outputs
-
-| Name | Description |
-|------|-------------|
-| buckets_list | The buckets list within the created project |
-
+[^]: (autogen_docs_start)
+
+## Inputs
+
+## Outputs
+
 [^]: (autogen_docs_end)
 
 ## Requirements
 ### Installation Dependencies
 - [Terraform](https://www.terraform.io/downloads.html) 0.11.x
 - [terraform-provider-google](https://github.com/terraform-providers/terraform-provider-google) plugin v1.12.0
-- [jq](https://stedolan.github.io/jq/)
 - [Python 2.7.x](https://www.python.org/getit/)
-- [terraform-docs](https://github.com/segmentio/terraform-docs/releases) 0.3.0
+- [terraform-docs](https://github.com/segmentio/terraform-docs/releases) (optional) 0.3.0
 
 ### Service Account
 In order to execute this module you must have a Service Account with the following roles assigned. There is a helpful setup script documented below which can automatically create this account for you.
 
-**Organization Roles**:
-- roles/resourcemanager.organizationAdmin
+### IAM Roles
+For this module to work, you need the following roles enabled on the Service Account.
 
-**Project Roles** on the Forseti install project:
-- roles/compute.instanceAdmin
-- roles/compute.networkViewer
-- roles/compute.securityAdmin
-- roles/deploymentmanager.editor
-- roles/iam.serviceAccountAdmin
-- roles/iam.serviceAccountUser
-- roles/serviceusage.serviceUsageAdmin
-- roles/storage.admin
+On the organization:

```

Comment:
@efouarge I recall that you had issues with Forseti when the user doesn't have an IAM role named `securityReviewer` or something - do you remember that as well? Do we need to mention that permission here?

Reply:
Updated the minimal required iam perms